comp/dogstatsd/server/server_test.go

```diff
@@ -940,59 +940,6 @@ func TestContainerIDParsing(t *testing.T) {
 	}
 }
 
-func testOriginOptout(t *testing.T, cfg map[string]interface{}, enabled bool) {
```

Comment:
Most of this test is obsolete after this refactoring, but it would be nice to have a test that ensures that we still extract origin from the various tags correctly.

Reply:
I've added this commit with more tests: [fixup! add server_test.go/TestOrigin](https://github.com/DataDog/datadog-agent/pull/23504/commits/ed18f00d3813f73ab9bff8e2aba0bfde6ae21c91) 👍